src/metadatad.cc

```diff
@@ -92,6 +93,9 @@ int main(int ac, char** av) {
 
   server.Start();
 
+  // Only register all the views after all the measures are initialized.
+  ::google::RegisterAllViewsForExport();
```

Comment:
I think this should be on L78 and should take care of initializing the (one) measure itself.

Reply:
I believe this follows from my earlier suggestion that the measures be encapsulated in the parts that report them, rather than stored in one global list of measures. I still think this is the right approach. However, is it possible to register the views on-demand just before exporting?

Reply:
Views should be created before you start recording due to the way aggregation works.

Reply:
as measure and view are created within single function, I think it is resolved.